tests/Schema/SchemaDiffTest.php

```diff
@@ -105,7 +105,7 @@ private function createPlatform(bool $unsafe)
         $platform->expects(self::exactly(1))
                 ->method('supportsSequences')
                 ->will(self::returnValue(true));
-        $platform->expects(self::exactly(2))
+        $platform->expects(self::atLeastOnce())
```

Comment:
Why does the test care how many times the `supportsForeignKeyConstraints()` method is invoked? If the change is necessary, please replace it with `self::any()`.